controllers/common.go

```diff
@@ -661,3 +661,58 @@ func (r *VolumeSnapshotRestoreReconciler) setVSRQueue(vsr *volsnapmoverv1alpha1.
 
 	return true, nil
 }
+
+func (r *VolumeSnapshotBackupReconciler) GetRepSrcPodSecurityContext(namespace string, sourcePVCName string) (*corev1.PodSecurityContext, error) {
```

Comment:
I see code duplication here, the function signatures and the logic is same for the both the functions:
```
func (r *VolumeSnapshotBackupReconciler) GetRepSrcPodSecurityContext(namespace string, sourcePVCName string) (*corev1.PodSecurityContext, error)

func (r *VolumeSnapshotRestoreReconciler) GetRepDestPodSecurityContext(namespace string, sourcePVCName string) (*corev1.PodSecurityContext, error)

```

Please combine them as one single function as the utility is same.